extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -606,7 +614,7 @@ export class AmpStoryEmbeddedComponent {
    */
   updateTooltipBehavior_(target) {
     if (matches(target, LAUNCHABLE_COMPONENTS['a'].selector)) {
-      addAttributesToElement(devAssert(this.tooltip_),
+      addAttributesToElement(dev().assertElement(devAssert(this.tooltip_)),
```

Comment:
This change seems to be everywhere in this PR. Is there a reason that `devAssert` doesn't just do what it's replaced with here?

Reply:
I don't believe `devAssert` guarantees a non-null `Element`. Without this change:
```js
extensions/amp-story/1.0/amp-story-embedded-component.js:617: ERROR - actual parameter 1 of addAttributesToElement$$module$src$dom does not match formal parameter
found   : (Element|null)
required: Element
      addAttributesToElement(devAssert(this.tooltip_),
                             ^^^^^^^^^^^^^^^^^^^^^^^^
```